x-pack/plugins/security_solution/cypress/integration/timelines/creation.spec.ts

```diff
@@ -131,6 +130,7 @@ describe('Create a timeline from a template', () => {
       cy.get(TIMELINE_FLYOUT_WRAPPER).should('have.css', 'visibility', 'visible');
       cy.get(TIMELINE_DESCRIPTION).should('have.text', getTimeline().description);
       cy.get(TIMELINE_QUERY).should('have.text', getTimeline().query);
+      closeTimeline();
```

Comment:
running 50 times in a row without:
![ss1](https://user-images.githubusercontent.com/6935300/135544222-4cac2214-f28f-4613-b75a-5631aabb506d.png)
 and with:
<img width="783" alt="Screen Shot 2021-09-30 at 4 05 06 PM" src="https://user-images.githubusercontent.com/6935300/135544216-9e6dea1e-a39f-4bb5-bc22-1043730e64fa.png">

Reply:
@MadameSheema it failed again on CI. can you please help? https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/156953/execution/node/536/log/?consoleFull